docs/core/SMS_Bridge_monitoring_spec_v2.2.md

```
@@ -0,0 +1,336 @@
+## 6. Ansible Integration
+
+### 6.1 Schema Reference
+
+**Important**: Grafana dashboard SQL queries must align with table definitions in `schema.sql`.
+
+When updating `schema.sql`:
+1. Update table names/columns in schema
+2. Update corresponding Grafana dashboard queries in Ansible
+3. Redeploy with `upgrade_sms_bridge_k3s.yml`
+
+### 6.2 ConfigMap Structure
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)